components/conference/ConferenceBase.module.less

```
@@ -0,0 +1,68 @@
+.base {
+  h2 {
+    font-size: 24px;
+  }
+  .container {
+    max-width: 1200px;
+    font-size: 14px;
+  }
+
+  .abstract {
+    p {
+      line-height: 1.6rem;
+    }
+  }
+
+  .review {
+    .review_item + .review_item {
+      margin-top: 40px;
+    }
+  }
+
+  .snap {
+    .media {
+      img {
+        width: 80px;
+      }
+    }
+  }
+
+  .animated {
+    animation: -webkit-animation 0.5s ease-in-out forwards;
+  }
+
+  @keyframes animated {
+    0% {
+      left: -50px;
+      opacity: 0.7;
+    }
+    100% {
+      left: 0;
+      opacity: 1;
+    }
+  }
+}
+
+@media screen and (max-width: 620px) {
+  .situation li {
+    display: -webkit-flex;
+    margin: 15px 0;
+  }
+  .snap .sm_flex_warped {
+    -webkit-flex-wrap: wrap;
+  }
+  .media {
+    width: 45%;
+    margin-top: 20px;
+    padding: 0 5px;
+  }
+  .media + .media {
+    margin-top: 20px;
+  }
+  .revImg {
+    width: 100%;
+  }
+  .media img {
+    width: 100px;
+  }
```

```suggestion
  .media {
    width: 45%;
    margin-top: 20px;
    padding: 0 5px;
    & + .media {
      margin-top: 20px;
    }
    img {
      width: 100px;
    }
  }
```
`revImg` 用 `w-100` 工具类替换